lib-ee/emqx_ee_connector/src/emqx_ee_connector_influxdb.erl

```diff
@@ -232,15 +236,14 @@ do_start_client(
     ClientConfig,
     Config = #{write_syntax := Lines}
 ) ->
+    Precision = maps:get(precision, Config, ms),
```

Comment:
Question:
is Precision here an atom like `ns` or binary like `<<"ns">>` ?

Reply:
atom

Reply:
It's converted to binary in `ClientConfig`, but `Config' holds atoms, I l've verified it with manual test and tracing.